app/Services/RestWrapper.js

```diff
@@ -36,7 +36,8 @@ class RestWrapper {
             .then((response) => {
                 return {
                     data: response().data,
-                    totalCount: response.headers()['x-total-count']
+                    totalCount: response().data.totalCount,
```

Comment:
All these checks are done in `admin-config` project, in the [ReadQueries](https://github.com/marmelab/admin-config/blob/master/lib/Queries/ReadQueries.js#L49). We should provide it the same structure in both ng-admin and react-admin.